pkg/store/bucket.go

```diff
@@ -314,6 +321,17 @@ func (s *BucketStore) SyncBlocks(ctx context.Context) error {
 		if err != nil {
 			return nil
 		}
+
+		inRange, err := s.isBlockInMinMaxRange(ctx, id)
```

Comment:
I can see us adding further methods of selection in the future, what do you think of wrapping this in a function like "keepBlock"?

Reply:
Sure SGTM, but maybe let's do this after we add more filters?

Reply:
I think we would probably end up with some kind of config file for store for specifying external labels and block time ranges. So this would be refactored to something more generic

Reply:
that works for me as well